src/lib/gridster.component.ts

```diff
@@ -76,6 +101,23 @@ export class GridsterComponent implements OnInit, OnChanges, OnDestroy, Gridster
     if (this.options.initCallback) {
       this.options.initCallback(this);
     }
+
+    if (this.options.selectable && this.options.selectable.enabled) {
+      this.selectionService.selectItem.subscribe((item: GridsterItemComponentInterface) => {
+        for (const x of this.grid) {
+          x.item.isSelected = false;
+          x.isSelected = false;
+        }
+        item.item.isSelected = true;
```

Comment:
`item.item` object is the object the user gives to the grid.
Don't think is a good idea to pollute that object with grid properties
Use `$item`